server/controller/test.js

```diff
@@ -1,6 +1,10 @@
+import Sequelize from 'sequelize';
 import models from '../models';
 
-const { Test, Question } = models;
+const { Op } = Sequelize;
```

Comment:
I think this is more readable `const { Op as Operator } = Sequelize;`

Reply:
Alright. I will make the changes